plugins/scaffolder-common/src/TaskSpec.ts

```diff
@@ -33,10 +39,25 @@ export type TemplateInfo = {
  * @public
  */
 export interface TaskStep {
+  /**
+   * A unqiue identifier for this step.
+   */
   id: string;
+  /**
+   * A display name to show the user.
+   */
   name: string;
+  /**
+   * The underlying action ID that will be called part of running this step.
+   */
   action: string;
+  /**
+   * Additional data that will be passed to the action.
+   */
   input?: JsonObject;
+  /**
+   * When this is false, or if the templated value string evaluates to something that is false the step will be skipped.
```

Comment:
```suggestion
   * When this is false, or if the templated value string evaluates to something that is falsy the step will be skipped.
```

is that more true? (heh)